src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  color: $main-color;
+  transition: all 300ms;
```

Comment:
Don't use transition: all, specify to what properties it should apply.
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/18de2aa8-496b-4a0a-a308-243c343ef129)